### DISCUSSION TITLE
Running containers lose permissions after some inactive time.

### DISCUSSION THREAD

**Discussion by theyousssef** [containers/podman#22294] [Q&A]
Hello,

I'm using Podman 4.6.1 and runc version 1.1.12 on WSL version: 2.1.5.0 with distribution: RHEL 8.9

I do run `podman exec -it <container name> /bin/bash`

it works fine, but after some time, when I run the same command, I get the following error:

`Error: runc: exec failed: unable to start container process: open /dev/ptmx: operation not permitted: OCI permission denied
`

I am not sure what causes this, I certainly don't do anything intentional here.

**Comment by GongT** [comment]
I got a similar error: `Error: crun: open /dev/ptmx: No such file or directory: OCI runtime attempted to invoke a command that was not found`

I got this error after some `podman rmi`

**Comment by rhatdan** [comment]
Please give us a repeater.  This looks like for some reason the /dev/ptmx from the host was removed.

We need podman info, what command were you running exactly when this happened.

Looks like your issue is very different from the original issue reported, since the original was a permission denied and you are gatting `not found`
